### DISCUSSION TITLE
How can I set the font on the title page of a reveal presentation?

### DISCUSSION THREAD

**Discussion by neelsmith** [quarto-dev/quarto-cli#9136] [Q&A] [answered]
### Description

I'm trying to set a font on the title of a revealjs presentation.  With html output like this document:

```qmd
---
title: "Debug font usage"
format: 
    html:
        theme: [default, custom.scss]    
---

## A regular slide
```

this SCSS works as I'd expect: title page and first slide have different fonts

```
/*-- scss:defaults --*/
@import url('https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap');

/*-- scss:rules --*/
.title {
    font-family: "Berkshire Swash", sans-serif;
}
```

But if I set the output type to `revealjs`, the `.title` class doesn't affect the font used on the title page.  Is that the wrong CSS class for reveal?

**Comment by mcanouil** [comment, marked as answer]
You need to use your browser inspect/developer mode to see what elements to targets, and you need to understand how CSS works such as nested rules, etc. (out of Quarto's scope)
This is true for every single HTML/(S)CSS customisation.


https://github.com/quarto-dev/quarto-cli/assets/8896044/d8bfc6f8-9788-4b46-8736-159b3dec8c74


````qmd
---
title: "Quarto Playground"
format: revealjs
include-in-header:
  text: |
    <style>
    #title-slide > .title {
      color: red;
    }
    </style>
---

This is a playground for Quarto.

````

**Comment by neelsmith** [comment]
Thanks -- I guess I hadn't properly scored the `.title` class in my css. Works now thanks to your help.
